src/simulation/m_global_parameters.fpp

```diff
@@ -487,7 +495,7 @@ contains
         mpp_lim = .false.
         time_stepper = dflt_int
         weno_eps = dflt_real
-        mapped_weno = .false.
+        teno_CT = 1d-6
```

Comment:
the default value should be `dflt_real` so that the code crashes if the user doesn't set this parameter but tries to use `teno`.

Reply:
My intention is for the code to run with teno without explicitly setting teno_CT, since the default value is most commonly used. However, I'm open to changing it if you believe it would improve consistency with the rest of the codebase.

Reply:
right now standard MFC practice is to require the user to set the variables that impact the method. for example, `weno_eps` should be required to be supplied by the user for "normal WENO." 

What you would do is have a statement that says something in `m_checker`. So we probably have something like 
```
if (weno_eps == dflt_real and weno_order > 1) then 
	print "weno is used by weno_eps is not set, aborting"
	stop
end if
```

and you would have something very similar for teno.

Reply:
note that checking for `==` on a float is bad programming practice, but i think we still do it (to my knowledge). if we do, do you mind opening a github issue on the topic? something like "replace checks against `==` for real valued variables with something resiliant to floating point errors".

Reply:
I see. I'll take the appropriate changes. Thanks!

Reply:
Yep I'll create the == issue

Reply:
> right now standard MFC practice is to require the user to set the variables that impact the method. for example, `weno_eps` should be required to be supplied by the user for "normal WENO."
> 
> What you would do is have a statement that says something in `m_checker`. So we probably have something like
> 
> ```
> if (weno_eps == dflt_real and weno_order > 1) then 
> 	print "weno is used, but weno_eps is not set. aborting"
> 	stop
> end if
> ```
> 
> and you would have something very similar for teno.

Actually this should say 
`print "weno is used, but weno_eps is not set. a typical value of weno_eps is 1e-6. aborting"`